eras/shelley/impl/cardano-ledger-shelley.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-shelley
-version:            1.8.0.0
+version:            1.8.0.1
```

Patch version is only updated when there are non-visible changes. So, if you ever have to create a lower bound like this `>=x.y.z.1` it means there was an issue with the version. patch version should never appear in the bounds.
```suggestion
version:            1.8.1.0
```

---

Ok, will update the part before  the patch then.